examples/clientcerts/clientcerts.values.yaml

```diff
@@ -0,0 +1,28 @@
+# These sections are used for the client side of certificate authentication.
+coderd:
+  clientTls:
```

Comment:
since there's already a coderd.tls section, maybe this could be coderd.tls.clientCertSecretName, coderd.tls.clientCertSecretKey to match?

Reply:
coderd.tlsClient.certSecret.name might be reasonable too, if you want to keep them grouped together. Having similar names just helps these come up in code completions

Reply:
@deansheather and @Emyrk discovered today that we're using TLS secrets for the other stuff, so we could do the same here (as with the way we configure DevURL things)

Reply:
Yesssss

Reply:
Changed this to use a TLS-type secret. So it's simply `coderd.clientTLS.secretName`.